packages/server/lib/project-base.ts

```diff
@@ -447,6 +447,10 @@ export class ProjectBase extends EE {
 
           await browsers.connectProtocolToBrowser({ browser: this.browser, foundBrowsers: this.options.browsers, protocolManager: this.protocolManager })
 
+          if (this.protocolManager.db) {
+            this.ctx.coreData.studio?.setProtocolDb(this.protocolManager.db)
```

Comment:
I wonder if there is a better way to do this since the StudioManager already has a reference to the protocolManager.